ruby_event_store_rom_sql/lib/ruby_event_store/rom/tasks/migration_tasks.rake

```diff
@@ -0,0 +1,17 @@
+
+desc 'Setup ROM EventRespository environment'
+task 'db:setup' do
+  ROM::SQL::RakeSupport.env = ::RubyEventStore::ROM.configure(ENV['DATABASE_URL'])
+end
+
+desc "Copy RubyEventStore SQL migrations to db/migrate"
```

Comment:
@joelvh are there any tests for this rake task?

Reply:
@paneq have you setup test for this before?

Reply:
@joelvh https://github.com/RailsEventStore/rails_event_store/blob/master/rails_event_store_active_record/spec/migration_generator_spec.rb

Reply:
Although I remember that there was lot's of pain involved @pawelpacana ?

Reply:
Later, in `bounded_context` we did something better:
- https://github.com/RailsEventStore/rails_event_store/blob/master/bounded_context/spec/spec_helper.rb
- https://github.com/RailsEventStore/rails_event_store/blob/master/bounded_context/spec/generators_spec.rb